apollo-router/src/plugins/cors.rs

```diff
@@ -321,6 +324,45 @@ impl<S> CorsService<S> {
             );
         }
 
+        // FIXME: Do we always want to do this, even for preflight requests?
```

Comment:
Reminder that this comment needs to be removed once we determine if we always want to add the headers.